code/modules/events/meteors.dm

```diff
@@ -1,44 +1,96 @@
 //meteor storms are much heavier
 /datum/event/meteor_wave
-	startWhen		= 6
-	endWhen			= 33
+	startWhen		= 86
+	endWhen			= 9999//safety value, will be set during ticks
+
+	var/wave_delay  = 13//Note, wave delay is in procs. actual time is equal to wave_delay * 2.1
+	var/min_waves 	= 11
+	var/max_waves 	= 16
+	var/min_meteors = 1
+	var/max_meteors = 2
+	var/duration = 420//Total duration in seconds that the storm will last after it starts
+
+
+	var/waves		= 8
+	var/next_wave 	= 86
 
 /datum/event/meteor_wave/setup()
-	endWhen = rand(15,30) * 3
+	startWhen += rand(-15,15)//slightly randomised start time
```

Comment:
yes, because startwhen is initialised at a value of 86. This is a modifier of +/- 15 added onto that, so the final result is 71-101

In case you wonder, event timers are done in procs, and as always a proc is 2.1 seconds. 86 procs is roughly 3 minutes